CppCoreGuidelines.md

```diff
@@ -14740,7 +14740,7 @@ Thread creation is expensive.
         // process
     }
 
-    void master(istream& is)
+    void scheduler(istream& is)
```

Comment:
FWIW, I think this is neither a master (in any of its multiple meanings) nor a scheduler. It extracts work items from input and sends them elsewhere for eventual execution. Maybe "dispatcher" or "message_processor"?

Reply:
@hsutter : any opinion on this? I have no favorite replacement.

Reply:
Personally I think any of the alternatives to "master" sound fine... how about "dispatcher"?

Reply:
"dispatcher" it is.